Assets/Scripts/Game/Main/ClientGameLoop.cs

```diff
@@ -562,6 +567,20 @@ public bool Init(string[] args)
         else
             m_StateMachine.SwitchTo(ClientState.Browsing);
 
+
+        VivoxSettings.chatSystemClient = m_ChatSystem;
+        VivoxSettings.targetServer = targetServer;
+        if (targetServer == "127.0.0.1")
```

Comment:
What is this doing?

Reply:
I need a the machines IP address to form a channel name. If multiple people are playing on their own machine using local host as the IP, there would be a overlap. I had to resolve localhost to a valid IP.

Reply:
In a survey of StackOverflow, this was my favorite simple solution.